core/java/iesi-core/src/main/java/io/metadew/iesi/script/execution/instruction/data/text/TextSubstring.java

```diff
@@ -36,7 +35,7 @@ public String generateOutput(String parameters) {
                 end += text.length()+1 ;
 
             verifyArguments(text, start, end);
-            return text.substring(start, end);
+            return text.replaceAll("\r\n|\r|\n","").substring(start, end);
```

Comment:
you cannot remove the newlines in the text being 'substringed'. There is a possibility the 'substringed' text contains a newline